### PR TITLE
More comprehensive wording on devices' wipe action button

### PR DESCRIPTION
**Pull request by Jerome-Herbinet**
Replace "Wipe all devices" with "Disconnect all devices and delete local data".
IMO, of course, it's longer, but the "Wipe all devices" wording lacks clarity on the action incurred.
So this new wording might be a better compromise.

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

**Review by szaimen — APPROVED** [review]
LGTM

**Review by nimishavijay — CHANGES_REQUESTED** [review]
Good idea to improve the wording! :) I am worried that "Disconnect all devices and delete local data" is quite long. How about "Delete data from all devices"? It provides more info than "Wipe all devices" (and probably translates better as well) and there is a confirmation modal to explain further anyway. What do you think? @nextcloud/designers

**Comment by Jerome-Herbinet** [comment]
> Good idea to improve the wording! :) I am worried that "Disconnect all devices and delete local data" is quite long. How about "Delete data from all devices"? It provides more info than "Wipe all devices" (and probably translates better as well) and there is a confirmation modal to explain further anyway. What do you think? @nextcloud/designers

@nimishavijay thanks for your comment :-)
Your suggestion "Delete data from all devices" is longer than "Wipe all devices" but does not contain some information : 
- it does not talk about disconnection (IMO, this half of the feature is very important to talk about)
- it does not talk about the "local" or "synced" side of data
So the problem remains the same, even if better words are used, but with a longer wording ;-)
Instead of "Disconnect all devices and delete local data", a little shorter wording would be "Disconnect devices and wipe local data". 
After this, I suggest improving the modal's description because it's not talking about disconnection. This could be done by myself with your suggestions in this same PR. 
I would also (still in this PR) improve the wording for apps' passwords UI in the "Security" section ("Wipe device").
What do you think ?

**Comment by nimishavijay** [comment]
Absolutely agreed about the modal! How about the item in the action menu is short and the modal has more info, like this: 

Action menu item:
- [ ] Change wording to "Delete data from all devices"
- [ ] Use the icon [phonelink-erase](https://fonts.google.com/icons?selected=Material%20Icons%3Aphonelink_erase%3A)
- [ ] Change the order of items in the action menu to better reflect the risk of the actions
  - [ ] Resend welcome email
  - [ ] Impersonate
  - [ ] Disable user
  - [ ] Delete data from all devices
  - [ ] Delete user

Modal:

```
Disconnect all devices and delete local data

In case of lost devices or exiting the organization, this will remotely wipe the Nextcloud data from all devices associated with [username] and disconnect the devices. Only works if the devices are connected to the internet.

[Cancel] [Wipe devices]

```

Things to note:
- [ ] Wording change in the heading of the modal
- [ ] Wording change in the description
- [ ] Wording change in the button (removed the username as the original text was `Wipe [username]'s devices`, and using an apostrophe ('s) may not translate well to other languages)

What do you think? Does this sufficiently communicate the result of the action? :)

**Review by nfebe — APPROVED** [review]
Another common way to say this for example in operating systems and hard disk manipulation software they often use "Erase device(s)" (Most times with a text that gives more details and of course a design that shows how dangerous the action is)
